www/base/frontend_proxy.js

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env node
+'use strict';
+
+const argv = require('minimist')(process.argv);
```

Comment:
I think this would be less hacky if we implemented this proxy in python aiohttp.
ex: https://github.com/oetiker/aio-reverse-proxy/blob/master/paraview-proxy.py

This would avoid the process index hack

Reply:
Thanks for suggestion, indeed python proxy would be better. Could we have the old implementation in the interim though?

Reply:
ok